vm/src/vm.rs

```diff
@@ -106,6 +106,12 @@ pub struct PySettings {
 
     /// sys.argv
     pub argv: Vec<String>,
+
+    /// Initialize VM after VM created
```

Comment:
This doc strings are definitely a good thing, but still I do not understand the logic of external initialization. Could you explain a bit more what this is and what it's effects are?

Reply:
The docstrings are modified and explain about initialization with the importers require an external file system or not.  Thanks for point out.  I am not truly understood the flow before, but it much clear now.